jni/offsets.c

```diff
@@ -349,11 +352,17 @@ static char* get_kernelver(char* str)
 
 struct offsets* get_offsets()
 {
-	char* devname = calloc(1, DEVNAME_LEN);
-	char* kernelver = calloc(1, KERNELVER_LEN);
+	char* devname, kernelver;
 	unsigned int i;
 	struct offsets* o = NULL;
 
+	devname = calloc(1, DEVNAME_LEN);
+	if (!devname)
```

Comment:
This is not necessary since `get_devname` and `get_kernelver` already check for null pointers.